pom.xml

```diff
@@ -12,7 +12,7 @@
     <name>VectorCAST Execution</name>
     <description>Run VectorCAST manage jobs from Jenkins</description>
     <artifactId>vectorcast-execution</artifactId>
-    <version>0.75-SNAPSHOT</version>
+    <version>0.76-SNAPSHOT</version>

```

Comment:
This should be reverted. The version will be bumped automatically during a maven release.

Reply:
Done...